examples/domainmodel/test/domainmodel-cli.test.ts

```diff
@@ -37,7 +37,7 @@ describe('Test the domainmodel CLI', () => {
     }
 
     test('Test action without CLI', async () => {
-        await generateAction(rawfileName, { destination, quiet: true });
+        await generateAction(rawfileName, { destination, quiet: false });
```

Comment:
Temporary, the quiet mode made it very difficult to actually see that there was an underlying error from vite otherwise. The plan is to switch that back.

Reply:
Latest commit adds a TODO as an additional reminder, just so that doesn't sneak in before merging.

Reply:
Underlying issue is resolved, so I've reverted this in an upcoming commit.